config/src/config/consensus_config.rs

```diff
@@ -241,25 +241,25 @@ impl Default for ConsensusConfig {
                     // Block enters the pipeline after consensus orders it, and leaves the
                     // pipeline once quorum on execution result among validators has been reached
                     // (so-(badly)-called "commit certificate"), meaning 2f+1 validators have finished execution.
-                    back_pressure_pipeline_latency_limit_ms: 800,
+                    back_pressure_pipeline_latency_limit_ms: 1000,
```

Comment:
Given our node is hitting 1s latency under expected load, I think this needs to be higher than 1s.

Reply:
Increased this limit further.